frontend/VolunteerOne/components/Reaction/Comment.js

```diff
@@ -118,8 +118,8 @@ const styles = StyleSheet.create({
       borderColor: "#32325D",
       backgroundColor: "#FFFFFF",
       borderRadius: 8,
-      width: 343,
-      marginLeft: -89,
+      width: 320,
```

Comment:
Just want to confirm that these dimensions still look good for posts without images, since they are two different components with different margins

Reply:
> Overall looks good. The input field hit box for the comment bar is still a bug needing to be fixed but I can address this in a later PR

Yeah the comment bar sensitivity is still something that needs to be addressed.

Reply:
> Just want to confirm that these dimensions still look good for posts without images, since they are two different components with different margins

Yeah, even with a post with no image it still looks good. Ill paste a screenshot of it below.
![Screen Shot 2023-04-30 at 4 55 51 PM](https://user-images.githubusercontent.com/58537683/235382059-d8cce727-533b-43a3-b427-cd1782b4baa4.png)